Sources/PrimerSDK/Classes/Core/3DS/ThreeDSecureService.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  ThreeDService.swift
+//  PrimerSDK
+//
+//  Created by Evangelos Pittas on 1/4/21.
+//
+
+#if canImport(UIKit)
+
+import Foundation
+import ThreeDS_SDK
+
+protocol ThreeDSecureServiceProtocol {
+    func threeDSecureBeginAuthentication(paymentMethodToken: PaymentMethodToken,
```

Comment:
[Nitpick]
Not sure the prefix `threeDSecure` is needed here. It's already in the protocol 🤔 
And this is not expected to be used by a delegate so... feels to me that `beginAuthentication` would be enough?

Reply:
Yep, will refactor